libs/dh/metering-point/feature-move-in/src/components/move-in.component.html

```diff
@@ -44,11 +44,11 @@
       [previousButtonLabel]="t('steps.contactDetails.label')"
       [nextButtonLabel]="t('save')"
     >
-      <dh-address-details-form
-        [addressDetailsForm]="addressDetailsForm"
-        (resetLegalForm)="emptyLegalAddressFormGroup()"
-        (resetTechnicalForm)="emptyTechnicalAddressFormGroup()"
-      />
+      <dh-address-details-form [addressDetailsForm]="addressDetailsForm"
+                               (resetLegalForm)="emptyLegalAddressFormGroup()"
+                               (resetTechnicalForm)="emptyTechnicalAddressFormGroup()"
+                               (pasteLegalFormDataIntoTechnicalForm)="pasteLegalFormDataIntoTechnicalForm()"
+                               (pasteTechnicalFormDataIntoLegalForm)="pasteTechnicalFormDataIntoLegalForm()"/>
```

Comment:
nitpick: this will probably be auto-formatted once the PR is merged in the base branch.